code/contracts/ONEWallet.sol

```diff
@@ -38,16 +39,16 @@ contract ONEWallet {
         height = height_;
         interval = interval_;
         t0 = t0_;
-        lifespan_ = lifespan_;
+        lifespan_ = lifespan_; // IH: what are the expected values?
         lastResortAddress = lastResortAddress_;
         dailyLimit = dailyLimit_;
-        maxOperationsPerInterval = maxOperationsPerInterval_;
+        maxOperationsPerInterval = maxOperationsPerInterval_; // IH: what are the expected values?
```

Comment:
I intend to use 1 for the initial release (to 10 users), and increase to 4 or 16 in subsequent releases.